### PR TITLE
Improve translation and fix orthographic and grammatical mistakes

### PR DESCRIPTION
**Pull request by clemalex824**
- Orthographic correction
- french instead of english terms

### CODE REVIEW

**Comment by clemalex824** [comment]
Error on Table of Contents : 
Can you explain me why readme must have 'Content' term instead of 'Sommaire' ?

File: readme.md [hunk 1]
```diff
@@ -5,15 +5,15 @@
 _Si vous êtes à l'aise avec l'anglais n'hésitez pas à consulter [l'Awesome list
 Home Assistant](https://www.awesome-ha.com/) initiée et hebergé par @frenck._
 
-## Contents
+## Sommaire
```
Comment (oncleben31) [inline readme.md:8]: `Contents` section is mandatory to be compliant with awesome list requirement and being in the future referenced in the [master awesome list ](https://github.com/sindresorhus/awesome#readme)

I know it's not ideal for having a full list in French

File: readme.md [hunk 2]
```diff
@@ -51,18 +51,18 @@ _Certains d'entre nous partagent leur configuration pour inspirer les autres. G
 
 ## Modules tiers utiles pour (ou par) des utilisateurs francophones
 
-_Home Assitant supporte dès la première installation de nombreux équipement ou services liés à la maison connecté. Vous pouvez compléter les intégrations disponibles avec des modules tiers._
+_Home Assitant supporte dès la première installation de nombreux équipement ou services liés à la maison connectée. Vous pouvez compléter les intégrations disponibles avec des modules tiers._
```
Comment (oncleben31) [inline readme.md:54]:
```suggestion
_Home Assitant supporte dès la première installation de nombreux équipements ou services liés à la maison connectée. Vous pouvez compléter les intégrations disponibles avec des modules tiers._
```

File: readme.md [hunk 3]
```diff
@@ -77,7 +77,7 @@ HACF propose un grand nombre de type de contenu pour fédérer la communauté fr
 
 ### Paradis Artificiels
 
-Le contenu de Paradis Artificiels est particulier. Ca a été un des premiers fédérateurs populaire de la communauté francophone de Home Assitant et nombreux d'entre nous en sont reconnaissants. Cependant nous n'avons plus aucune nouvelles depuis plus d'un an et les outils qu'il a mis en place ne peuvent plus être maintenus comme il faudrait.
+Le contenu de Paradis Artificiels est particulier. Ca a été un des premiers fédérateurs populaire de la communauté francophone de Home Assitant et nombreux d'entre nous en sont reconnaissants. Cependant nous n'avons plus aucunes nouvelles depuis début 2019 et les outils qu'il a mis en place ne peuvent plus être maintenus comme il le faudrait.
```
Comment (oncleben31) [inline readme.md:80]:
```suggestion
Le contenu de Paradis Artificiels est particulier. Ca a été un des premiers fédérateurs populaire de la communauté francophone de Home Assitant et nombreux d'entre nous en sont reconnaissants. Cependant nous n'avons plus aucune nouvelle depuis début 2019 et les outils qu'il a mis en place ne peuvent plus être maintenus comme il le faudrait.
```

**Review by oncleben31 — CHANGES_REQUESTED** [review]
Thank you for taking time to fix typos and grammatical mistakes. 
Some corrections to be almost perfect ;-)

**Review by oncleben31 — APPROVED** [review]

File: readme.md [hunk 4]
```diff
@@ -5,15 +5,15 @@
 _Si vous êtes à l'aise avec l'anglais n'hésitez pas à consulter [l'Awesome list
 Home Assistant](https://www.awesome-ha.com/) initiée et hebergé par @frenck._
 
-## Contents
+## Contents (Table des matières)
```
Comment (oncleben31) [inline readme.md:8]:
```suggestion
## Contents
```

**Review by oncleben31 — APPROVED** [review]

**Comment by oncleben31** [comment]
Thank you @clemalex824

**Pull request merged**